<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WGuest</class>
 <widget class="QWidget" name="WGuest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Guest info</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="7" column="2">
      <widget class="EQLineEdit" name="leNationality">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Code</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Date of birth</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>First name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Last name</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Sex</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Tel. 1</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Passport</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Post box</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Address</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Tel. 2</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Country of resident</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Nationality</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Remarks</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="EQLineEdit" name="leNationalityCode">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="ShowButton" stdset="0">
        <bool>true</bool>
       </property>
       <property name="Field" stdset="0">
        <string>f_nation</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="EQLineEdit" name="leGuestCode">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="Field" stdset="0">
        <string>f_id</string>
       </property>
       <property name="PrimaryKey" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="EQLineEdit" name="leTitle">
       <property name="ShowButton" stdset="0">
        <bool>true</bool>
       </property>
       <property name="Field" stdset="0">
        <string>f_title</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="EQLineEdit" name="leFirstname">
       <property name="Field" stdset="0">
        <string>f_firstName</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="EQLineEdit" name="leLastname">
       <property name="Field" stdset="0">
        <string>f_lastName</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="EQComboBox" name="cbSex">
       <property name="Field" stdset="0">
        <string>f_sex</string>
       </property>
       <property name="Cache" stdset="0">
        <number>52</number>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="EQLineEdit" name="lePlaceBirth">
       <property name="Field" stdset="0">
        <string>f_placeBirth</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="2">
      <widget class="EQLineEdit" name="lePassport">
       <property name="Field" stdset="0">
        <string>f_passport</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="2">
      <widget class="EQLineEdit" name="leAddress">
       <property name="Field" stdset="0">
        <string>f_address</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1" colspan="2">
      <widget class="EQLineEdit" name="leTel1">
       <property name="Field" stdset="0">
        <string>f_tel1</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1" colspan="2">
      <widget class="EQLineEdit" name="leTel2">
       <property name="Field" stdset="0">
        <string>f_tel2</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1" colspan="2">
      <widget class="EQLineEdit" name="leEmail">
       <property name="Field" stdset="0">
        <string>f_email</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1" colspan="2">
      <widget class="EQLineEdit" name="lePostBox">
       <property name="Field" stdset="0">
        <string>f_postBox</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1" colspan="2">
      <widget class="EQTextEdit" name="teRemarks">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>75</height>
        </size>
       </property>
       <property name="Field" stdset="0">
        <string>f_remarks</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0" colspan="4">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnRemove">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnOk">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="EDateEdit" name="deBirth">
       <property name="Field" stdset="0">
        <string>f_dateBirth</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>EDateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">edateedit.h</header>
  </customwidget>
  <customwidget>
   <class>EQTextEdit</class>
   <extends>QTextEdit</extends>
   <header location="global">eqtextedit.h</header>
  </customwidget>
  <customwidget>
   <class>EQComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">eqcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leTitle</tabstop>
  <tabstop>leFirstname</tabstop>
  <tabstop>leLastname</tabstop>
  <tabstop>cbSex</tabstop>
  <tabstop>deBirth</tabstop>
  <tabstop>lePlaceBirth</tabstop>
  <tabstop>leNationalityCode</tabstop>
  <tabstop>lePassport</tabstop>
  <tabstop>leAddress</tabstop>
  <tabstop>leTel1</tabstop>
  <tabstop>leTel2</tabstop>
  <tabstop>leEmail</tabstop>
  <tabstop>lePostBox</tabstop>
  <tabstop>teRemarks</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>leNationality</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
